packages/issuer-sdk-js/src/crypto.ts

```diff
@@ -0,0 +1,18 @@
+import * as base64 from "@stablelib/base64";
+import * as utf8Codec from "@stablelib/utf8";
+import nacl from "tweetnacl";
+
+export async function encrypt(message: string, encryptionPublicKey: string, secretKey: string) {
+  const nonce = nacl.randomBytes(nacl.box.nonceLength);
+  const encrypted = nacl.box(
```

Comment:
Please keep the error description we have in the original code: https://github.com/idos-network/idos-sdk-js/blob/4a94d48dd3c8ed841b45889610a7b907b1a6b972/apps/idos-enclave/src/lib/enclave.js#L181